src/dos/program_keyb.cpp

```diff
@@ -20,112 +20,385 @@
 	        "Usage:\n"
-	        "  [color=light-green]keyb[reset] [color=light-cyan][LANGCODE][reset]\n"
-	        "  [color=light-green]keyb[reset] [color=light-cyan]LANGCODE[reset] [color=white]CODEPAGE[reset] [color=white][CODEPAGEFILE][reset]\n"
+	        "  [color=light-green]keyb[reset]\n"
+	        "  [color=light-green]keyb[reset] /list\n"
+	        "  [color=light-green]keyb[reset] [color=light-cyan]LAYOUT[reset] [[color=white]CODEPAGE[reset]] /rom\n"
+	        "  [color=light-green]keyb[reset] [color=light-cyan]LAYOUT[reset] [[color=white]CODEPAGE[reset] [[color=white]CPIFILE[reset]]]\n"
 	        "\n"
 	        "Parameters:\n"
-	        "  [color=light-cyan]LANGCODE[reset]      language code or keyboard layout ID\n"
-	        "  [color=white]CODEPAGE[reset]      code page number, such as [color=white]437[reset] and [color=white]850[reset]\n"
-	        "  [color=white]CODEPAGEFILE[reset]  file containing information for a code page\n"
+	        "  [color=light-cyan]LAYOUT[reset]    keyboard layout code\n"
+	        "  [color=white]CODEPAGE[reset]  code page number, such as [color=white]437[reset] or [color=white]850[reset]\n"
+	        "  [color=white]CPIFILE[reset]   screen font file, in CPI or CPX format\n"
+	        "  /list     displays available keyboard layout codes\n"
+	        "  /rom      if possible, uses screen font from display adapter ROM\n"
 	        "\n"
 	        "Notes:\n"
-	        "  Running [color=light-green]keyb[reset] without an argument shows the currently loaded keyboard layout\n"
-	        "  and code page. It will change to [color=light-cyan]LANGCODE[reset] if provided, optionally with a\n"
-	        "  [color=white]CODEPAGE[reset] and an additional [color=white]CODEPAGEFILE[reset] to load the specified code page\n"
-	        "  number and code page file if provided. This command is especially useful if\n"
-	        "  you use a non-US keyboard, and [color=light-cyan]LANGCODE[reset] can also be set in the configuration\n"
-	        "  file under the [dos] section using the \"keyboardlayout = [color=light-cyan]LANGCODE[reset]\" setting.\n"
+	        "  - Running [color=light-green]keyb[reset] without an argument shows the currently loaded keyboard layout\n"
+	        "    and code page.\n"
+	        "  - The [color=white]CPIFILE[reset], if specified, must contain screen font for the given [color=white]CODEPAGE[reset].\n"
+	        "  - If no custom [color=white]CPIFILE[reset] is specified, it tries to find a suitable screen font\n"
+	        "    in one of the bundled CPI files.\n"
+	        "  - If [color=white]CODEPAGE[reset] is not specified, it tries use the screen font from the display\n"
+	        "    adapter ROM first, just like when using /rom switch.\n"
+	        "  - Only EGA or better display adapters allow to change the screen font; MDA,\n"
+	        "    CGA, or Hercules always use the ROM screen font.\n"
+	        "  - The 'us' keyboard layout can be used with any code page; all the other\n"
```

Comment:
Avoid passive structures:

"You can use the 'us' keyboard layout with any code page"

Reply:
OK, changed.